lib/content-services/src/lib/viewer/components/alfresco-viewer.component.spec.ts

```diff
@@ -152,15 +147,7 @@ describe('AlfrescoViewerComponent', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [
-                AuthModule.forRoot({ useHash: true }),
-                TranslateModule.forRoot(),
-                MatButtonModule,
-                MatIconModule,
-                MatDialogModule,
-                HttpClientTestingModule,
-                ...VIEWER_DIRECTIVES
-            ],
+            imports: [AuthModule.forRoot({ useHash: true }), MatDialogModule, NoopTranslateModule, ...VIEWER_DIRECTIVES],
```

Comment:
Missing imports for MatButtonModule and MatIconModule. Is that intentional?

Reply:
Yes, the standalone components have all their imports inside, I am cleaning the useless imports